experiment/simulation/js/main.js

```diff
@@ -1 +1,148 @@
-//Your JavaScript goes in here
+import data from "./data.js";
+
+const setMolecule = (angle, animate = false) => {
+  let width = 400;
+  if (window.innerWidth < 900) {
+    width = window.innerWidth;
+  }
+  let movie = new ChemDoodle.MovieCanvas3D("movie", width, 400);
```

Comment:
width and movie both can be const. Use const where possible, use let only when you have to